common/gsutil.py

```diff
@@ -19,29 +19,29 @@
 logger = logs.Logger('gsutil')
 
 
-def gsutil_command(arguments, *args, parallel=False, **kwargs):
+def command(arguments, *args, parallel=False, **kwargs):
```

Comment:
Leave name as is and make this function private.

Reply:
This will be called by test functions.

Reply:
Basically, I want rename the gsutil_command() to command(). Because 
- this function is already inside the gsutil module. There shouldn't be some other *_command things in the to use in the gsutil module.
- if we want to maintain a unified format for upper level call, then `command()` is more general. Because `local_utils`  module can also have `command()`, but cannot have `gsutil_command()`.